sympy/printing/pycode.py

```diff
@@ -128,6 +128,12 @@ def _print_sign(self, e):
         return '(0.0 if {e} == 0 else {f}(1, {e}))'.format(
             f=self._module_format('math.copysign'), e=self._print(e.args[0]))
 
+    def _print_NegativeInfinity(self, expr):
+        return "float('-inf')"
+
+    def _print_ComplexInfinity(self, expr):
+        return "float('nan')"
```

Comment:
I was wondering if it would be a better decision to actually call this NaN printing here https://github.com/ArighnaIITG/sympy/blob/1e74dfeb8ff522918df28c123c9f85538a5a49fe/sympy/printing/pycode.py#L121 ?

Reply:
True. Function calls are always better than redundant code statements.
Implemented. Thank you!!